python/langsmith/async_client.py

```diff
@@ -43,7 +70,9 @@ def __init__(
             ]
         ] = None,
         retry_config: Optional[Mapping[str, Any]] = None,
+        info: Optional[Union[dict, ls_schemas.LangSmithInfo]] = None,
         web_url: Optional[str] = None,
+        auto_batch_tracing: bool = False,
```

Comment:
info should probably go after web_url to avoid a breaking change in positional args. maybe even better, we could start enforcing that args with default vals need to be named:

```suggestion
        web_url: Optional[str] = None,
        *,
        info: Optional[Union[dict, ls_schemas.LangSmithInfo]] = None,
        auto_batch_tracing: bool = False,
```

Reply:
Good point, thank you, will update!